src/gap.c

```diff
@@ -1203,10 +1203,9 @@ Obj FuncWindowCmd (
 *F  FuncDownEnv( <self>, <level> )  . . . . . . . . .  change the environment
 */
 
-/*
-TL: Obj  ErrorLVars0;
-TL: Obj  ErrorLVars;
-TL: Int  ErrorLLevel;
+/* TL: Obj  ErrorLVars0;    
+   TL: Obj  ErrorLVars;
+   TL: Int  ErrorLLevel;
 */
```

Comment:
Is this part of the change really necessary? Seems to serve no purpose, but it'll break my PR #934 (which I'd then have to rebase once again). So, if you don't mind terribly, perhaps you could drop this one change? I'd then be happy to merge the rest as it is :-).
Thanks!